src/ProgressBar.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import theme from './theme'
+import {COMMON, get} from './constants'
+
+const Bar = styled.span`
+  width: ${props => (props.progress ? `${props.progress}%` : 0)};
+  ${COMMON}
+`
+
+const ProgressContainer = styled.span`
+  display: flex;
+  height: ${get('spacing.2')};
+  overflow: hidden;
+  background-color: ${get('colors.gray.2')};
+  border-radius: ${get('radii.1')}px;
+  height: ${props => (props.large ? '10px' : '5px')};
```

Comment:
I think this one should be 8px, right?
```suggestion
  height: ${props => (props.large ? '10px' : '8px')};
```

Reply:
Just realized that 5px is what is used on the issues page - so if we want to stick to just a `large` prop, then we should probably leave this as is.

Reply:
This is resolved in my sizing api refactor commit